package-system/AWSNativeSDK/FindAWSNativeSDK.cmake.Windows

```diff
@@ -251,62 +281,21 @@ ly_declare_aws_library(
 #########
 ######### Grouping Definitions #########
 #########
-
-
-#### Dependencies ####
-ly_declare_aws_library(
-    NAME 
-        Dependencies
-    BUILD_DEPENDENCIES 
-        3rdParty::AWSNativeSDK::Checksums
-        3rdParty::AWSNativeSDK::Common
-        3rdParty::AWSNativeSDK::EventStream
-)
-
-#### IdentityMetrics ####
-ly_declare_aws_library(
```

Comment:
Curious: What's the reason to remove the IdentityMetrics and IdentityLambda libraries? Are they not in use somewhere in the engine?

Reply:
correct, they are not used anywhere